api_tests/users/serializers/test_serializers.py

```diff
@@ -58,3 +46,26 @@ def test_user_serializer(self, user):
         assert 'institutions' in relationships
         assert 'preprints' in relationships
         assert 'registrations' in relationships
+
+    def test_user_serializer_with_related_counts(self, user):
+        req = make_drf_request_with_version(version='2.0')
+        req.query_params['related_counts'] = True
+        result = UserSerializer(user, context={'request': req}).data
+        data = result['data']
+
+        # Relationships
+        relationships = data['relationships']
+        assert relationships['quickfiles']['links']['related']['meta']['count'] == 0
+        assert relationships['nodes']['links']['related']['meta']['count'] == 0
+        assert relationships['institutions']['links']['related']['meta']['count'] == 0
+        assert relationships['preprints']['links']['related']['meta']['count'] == 0
+        assert relationships['registrations']['links']['related']['meta']['count'] == 0
```

Comment:
I'd like to see additional tests besides the serializer tests,  for preprint, registrations, and node counts, at minimum.  With the creator being `user_1` say, make a few preprints, registrations, and nodes in a variety of states (published, not published, public, deleted, approved, unapproved, for the corresponding models).  I'd then check related_counts are accurate when logged in as both `user_1`, non contrib `user_2`, and unauthenticated.